app/views/like/_like.html.erb

```diff
@@ -81,7 +86,26 @@
     
         <br />
         <hr style='clear:both;' />
-    
+        <div>
+            <span>
+                <% if node.type == 'note' %>
+                    <%= number_with_delimiter(@node.views) %> <%= t('notes.show.views') %>
```

Comment:
Aha - here, i think we just need `node.views` and not `@node.views`! I'm going to try making this change to see if it works!!